crates/zoon/src/element/ability/has_lang.rs

```diff
@@ -0,0 +1,7 @@
+use crate::*;
+
+pub trait HasLang: UpdateRawEl + Sized {
+    fn lang<'a>(self, lang: impl IntoCowStr<'a>) -> Self {
+        self.update_raw_el(move |raw_el| raw_el.lang(lang))
+    }
+}
```

Comment:
Please add a trailing newline (ideally enable automatic adding in your IDE).

Also make sure you run `cargo fmt --all` ideally as a part of all commits or at least in the last commit.